Factors.py

```diff
@@ -0,0 +1,11 @@
+number=input("Enter the number:")
+factors = []
+for i in range(1,int(number)):
```

Comment:
loop does not cover the entered number, for i in range (1,int(number+1)) should fix it.

Reply:
fixed